docs/search.md

```diff
@@ -0,0 +1,45 @@
+Search
+======
+
+The search endpoint provides a powerful way of finding items that are accessible by a single user or an entire 
+enterprise. Leverage the parameters listed below to generate targeted advanced searches.
+
+Search for Content
+------------------
+
+To get a list of items matching a serch query, call [`search.query(query, limit=None, offset=0, **kwargs)`][query] will return an `Iterable` that allows you 
+to iterate over the [`Item`][item_class] objects in the collection.
+
+```python
+items = client.search().query(query='TEST QUERY', limit=100, file_extensions=['pdf', 'doc'])
+for item in items:
+    print('The item ID is {} and the item name is {1}'.format(item.id, item.name)
```

Comment:
Missing the closing paren for `print()` here.  Also, both placeholders should be numbered